core-bundle/assets/controllers/tinymce-controller.js

```diff
@@ -0,0 +1,15 @@
+import { Controller } from '@hotwired/stimulus';
+
+export default class extends Controller {
+    static values = {
+        config: String
```

Comment:
You can use the [`Object`](https://stimulus.hotwired.dev/reference/values#types) type here, then you don't have to execute `JSON.parse()` yourself :)

Reply:
Done in 99b99b1